Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -514,6 +514,44 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     return .visitChildren
   }
 
+  public override func visit(_ node: CanImportExprSyntax) -> SyntaxVisitorContinueKind {
+    if shouldSkip(node) {
+      return .skipChildren
+    }
+
+    if let label = node.label,
+      label.presence == .missing
+    {
+      addDiagnostic(
+        label,
+        .canImportWrongSecondParameterLabel,
+        handledNodes: [label.id]
+      )
+
+      [node.unexpectedBetweenlabelAndcolon?.id, node.colon?.id, node.versionTuple?.id].compactMap { $0 }.forEach { handledNodes.append($0) }
+    }
+
+    if let versionTuple = node.versionTuple,
+      let unexpectedVersionTuple = node.unexpectedBetweenversionTupleAndRightParen
+    {
+      if versionTuple.major.presence == .missing {
+        addDiagnostic(
+          versionTuple,
+          CannotParseVersionTuple(versionTuple: unexpectedVersionTuple),
+          handledNodes: [versionTuple.id, unexpectedVersionTuple.id]
+        )
+      } else {
+        addDiagnostic(
+          unexpectedVersionTuple,
+          TrailingVersionAreIgnored(version: versionTuple),
```

Comment:
Should we just consume any trailing components of a version in `parseVersionTuple` and have a generalized diagnostic that applies to all version tuples everywhere?